src/apps/components/AppDetailsPage/AppAdditionalInfo/index.ts

```diff
@@ -0,0 +1 @@
+export { AppAdditionalInfo } from "./AppAdditionalInfo";
```

Comment:
**Question:** why we exported this file? 🤔 

I mean it was causing PW to fail? 🤔

Reply:
@krzysztofzuraw fixed; I've added this by accident - thank you for pointing it out!